__tests__/integration.tests.js

```diff
@@ -103,6 +103,17 @@ describe("/api/articles/:article_id", () => {
                 expect(msg).toBe("Not found");
             });
     });
+    describe("FEATURE: GET /api/articles/:article_id (comment_count)", () => {
+        test('GET: 200, should respond with an array containing a single object with given id and correct keys', () => {
+            return request(app)
+                .get("/api/articles/2")
+                .expect(200)
+                .then((response) => {
+                    const { article } = response.body;
+                    expect(article).toHaveProperty("comment_count");
+                });
```

Comment:
You can also check the value of comment count as we know how many comments article 2 has. toHaveProperty can take a second optional value argument to compare the received property value.

Reply:
Thanks Hannah. Upon doing this i realised i should probably be casting the comment_count as an INT anyways so ill remember to check that next time!